examples/README.md

```diff
@@ -29,3 +29,27 @@ This ASP.NET web application is a .NET Framework MVC 5 app. It leverages the [co
 
 ## [WebFormApp (.NET Framework)](./DotNetFramework/WebFormApp)
 This ASP.NET web application is a .NET Framework Web Forms app. It demonstrates how to leverage the App Configuration [.NET Standard provider library](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration/) to achieve dynamic configuration and control feature launches with feature flags. The same technique applies to .NET Framework MVC apps.
+
+## [Spring Samples](https://github.com/Azure-Samples/azure-spring-boot-samples/)
+
+Contains multiple Azure App Configuration Spring boot sample projects.
+
+### azure-appconfiguration-sample
+
+This example shows how to use the basic features of Azure App Configuration in a in Java Spring cloud application.
+
+### azure-appconifugration-refresh-sample
+
+This example shows how to dynamically update application settings in a Spring Web application.
```

Comment:
Is "application settings" a concept in a Spring app, not properties or something?

Reply:
Application Setting is the term we use in MS docs and that Alex has been having me use.

Reply:
If there is a native term to Spring apps, we should use that. We should talk to Spring developers in their language instead of forcing them to learn ours.

Reply:
Please terminologies that are familiar to Spring developers.